go.mod

```diff
@@ -164,7 +164,7 @@ replace (
 	// See https://github.com/tecbot/gorocksdb/pull/216
 	github.com/tecbot/gorocksdb => github.com/cosmos/gorocksdb v1.1.1
 
-	github.com/tharsis/ethermint => github.com/crypto-org-chain/ethermint v0.10.0-alpha1-cronos-7
+	github.com/tharsis/ethermint => github.com/yihuang/ethermint v0.10.0-alpha1-cronos-6.0.20220322095201-0abd55fe59f7
```

Comment:
https://github.com/yihuang/ethermint/compare/v0.10.0-alpha1-cronos-7...0abd55fe59f7?expand=1

Reply:
```go
txHash := common.BytesToHash(tmtypes.Tx(data.Tx).Hash())
```
does this get cosmos tx hash?